pages/place/formatters/index.jsx

```diff
@@ -1,15 +1,13 @@
 import React, { Fragment } from 'react';
-import { defineValue, joinAcronyms, labelFromCode } from '../../common/formatters';
+import { joinAcronyms, labelFromCode } from '../../common/formatters';
 
 export default {
   suitability: {
     format: joinAcronyms,
-    label: defineValue,
     mapOptions: labelFromCode
   },
   holding: {
     format: joinAcronyms,
-    label: defineValue,
```

Comment:
What is the reasoning behind removing these? It seems unrelated to the issue at hand.

Reply:
We changed the TableHeader component so that if the schema has a label prop present for a column, it tries to render it - in this case it would try and render the function without calling it, resulting in an error.

I'm not really sure what these were for - removing it doesn't change the header text (Suitability / Holding) because they fall back to using the content files.

Reply:
Thanks.